src/rgw/rgw_lua.cc

```diff
@@ -213,11 +213,33 @@ int install_packages(const DoutPrefixProvider *dpp, rgw::sal::Driver* driver,
     return rc;
   }
   install_dir.assign(tmp_luarocks_path);
+  
+  {  
+    bp::ipstream is;
+    const auto cmd = p.string() + " config";
+    bp::child c(cmd, bp::std_in.close(), (bp::std_err & bp::std_out) > is);
+    
+    std::string lines = std::string("Lua CMD: ") + cmd;
+    std::string line;
+    
+    do {
+      if (!line.empty()) {
+        lines.append("\n\t");
+        lines.append(line);
+      }
+    } while (c.running() && std::getline(is, line));
+
+    c.wait();
+    line = "exit code: " + std::to_string(c.exit_code());
+    lines.append("\n\t");
+    lines.append(line);
+    ldpp_dout(dpp, 20) << lines << dendl;
```

Comment:
if the only purpose of this stuff is to log the output at debug_rgw 20, it's worth making the whole block conditional on:
```c++
if (cct->_conf->subsys.should_gather<ceph_subsys_rgw, 20>()) {
```